<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsScreen</class>
 <widget class="QWidget" name="SettingsScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>68</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>419</width>
    <height>68</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>419</width>
    <height>68</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(110, 110, 110, 255), stop:0.101942 rgba(91, 91, 91, 255), stop:0.820388 rgba(60, 60, 60, 255), stop:0.990291 rgba(37, 37, 37, 255));
	color: rgb(225, 225, 225);
}
QTextBrowser, QLineEdit {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(37, 37, 37, 147), stop:0.0873786 rgba(60, 60, 60, 150), stop:0.932039 rgba(91, 91, 91, 148), stop:1 rgba(110, 110, 110, 148));
	border: 1px  solid rgb(147, 45, 203);
	border-radius: 3px;
}

QLCDNumber, QSpinBox, QDateTimeEdit {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(37, 37, 37, 147), stop:0.0873786 rgba(60, 60, 60, 150), stop:0.932039 rgba(91, 91, 91, 148), stop:1 rgba(110, 110, 110, 148));
	border: 1px  solid rgb(44, 44, 44);
	border-radius: 3px;
}

QLabel {
	background-color:rgba(255, 255, 255, 0);
}

QPushButton {
	background-color:rgba(255, 255, 255, 0);
}

QProgressBar {
	background-color: rgb(44, 44, 44);
	border: 1px  solid rgb(44, 44, 44);
	margin: 8px, 8px;
	color:rgba(255, 255, 255, 0);
}
QProgressBar::chunk {
	background-color: rgb(147, 45, 203);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Tags:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TagsSet">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MinSet">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="suffix">
    <string> min</string>
   </property>
   <property name="maximum">
    <number>4320</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Until:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DoneSet">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="TimeSet">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="date">
    <date>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>d.MMM.yy h:mm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
